### ISSUE TITLE
Hyperparams for state of the art performance

### ISSUE THREAD

**Issue by brando90** [StefOe/all-conv-pytorch#1]
Hi, I was looking at ur other file with the hyper params etc but u didn't show it achieving 95% accuracy...I was wondering, did you ever manage to achieve that?

**Comment by yMayanand** [comment]
> 
> 
> Hi, I was looking at ur other file with the hyper params etc but u didn't show it achieving 95% accuracy...I was wondering, did you ever manage to achieve that?

i have also tried this architecture but i couldnt manage to reduce loss for cifar10 dataset.
Loss is not decreasing any further.
@brando90  could please tell how did you managed to get accuracy of 95 %,
